libs/pq_proto/src/lib.rs

```diff
@@ -672,6 +675,18 @@ pub struct WalSndKeepAlive {
     pub request_reply: bool,
 }
 
+#[derive(Debug)]
+pub struct InterpretedWalRecordsBody<'a> {
+    /// End of raw WAL in [`Self::data`]
+    pub streaming_lsn: u64,
+    /// Current end of WAL on the server
+    pub safekeeper_wal_end_lsn: u64,
```

Comment:
nit: can we call this `commit_lsn` or something? I had to go all the way over to the Pageserver side to understand why this was of any interest, I feel like `commit_lsn` makes that purpose clearer.

Reply:
Sure, done in https://github.com/neondatabase/neon/pull/9746/commits/1c8f4938cb8d860a91ed1eb7bacb85df79fbdf9d. Also updated the `rust-postgres` PR.